tests/test_variant.py

```diff
@@ -66,12 +67,26 @@ def test_samples_setter(self, line, sample_names, new_value, expected_output):
         ('chr1	726	.	G	C,T	500	.	DP=200;MQ=250.00	GT:AD:AF:DP:GQ	0/1:10,160,30:0.8,0.15:200:420', ['proband'], Zygosity.heterozygote),
         ('chr1	726	.	G	C,T	500	.	DP=200;MQ=250.00	GT:AD:AF:DP:GQ	1/1:10,160,30:0.8,0.15:200:420', ['proband'], Zygosity.homozygote),
         ('chr1	726	.	G	C,T	500	.	DP=200;MQ=250.00	GT:AD:AF:DP:GQ	0/0:10,160,30:0.8,0.15:200:420', ['proband'], Zygosity.reference),
+        ('chrX	726	.	G	C,T	500	.	DP=200;MQ=250.00	GT:AD:AF:DP:GQ	1:10,160,30:0.8,0.15:200:420', ['proband'], Zygosity.homozygote),
+        ('chrM	726	.	G	C,T	500	.	DP=200;MQ=250.00	GT:AD:AF:DP:GQ	.:10,160,30:0.8,0.15:200:420', ['proband'], Zygosity.no_coverage),
```

Comment:
I suggest adding a few more test cases:
* where the ALT contains more than 9 values (so that the x/y will have length more than 3)
* where there are 3 ALT values and the GT is something like 1/2
* three cases with a dot in different places in the diploid call: maybe ./1, 0/., ./.
* GT=0 on sex chromosome